src/components/refactor/dialogs/connectivity/connectivity-form-utils.js

```diff
@@ -0,0 +1,133 @@
+/**
+ * Copyright (c) 2022, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+import yup from '../../utils/yup-config';
```

Comment:
We can use the absolute import: 
Add the json below in the file named **jsconfig.json** in the root : 
```
{
    "compilerOptions": {
      "baseUrl": "src"
    },
    "include": ["src"]
  } 
```

and remplace this import with `import yup from 'components/refactor/utils/yup-config';`

https://saurabhshah23.medium.com/react-app-with-absolute-paths-using-jsconfig-json-2b07b1cb24d4